neo4j-to-neptune/pom.xml

```diff
@@ -162,6 +162,70 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <!-- JaCoCo Code Coverage -->
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.8.12</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>prepare-agent</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>report</id>
+                        <phase>test</phase>
+                        <goals>
+                            <goal>report</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <!-- SpotBugs Static Analysis -->
+            <plugin>
+                <groupId>com.github.spotbugs</groupId>
+                <artifactId>spotbugs-maven-plugin</artifactId>
+                <version>4.7.3.0</version>
```

Comment:
Any reason we're using these specific plugin versions? They seem to be a bit outdated. https://mvnrepository.com/artifact/com.github.spotbugs/spotbugs-maven-plugin

I don't think it's that big of a deal if we can't use the newer versions, just curious.